### ISSUE TITLE
NavMain disappears in BrandHeader and React

### ISSUE THREAD

**Issue by tommy-codez** [telekom/scale#546]
Hi there,

we want to use the Brand Header & Navigation Bar with React. My expectation was that the main navigation Bar stays on top during scrolling. 
When I scroll down the bar gets extended at a certain point, showing the logo and the menu options but doesnt stay on top when I scroll further. 

Do I need something special that it stays on top ? Here my simple example code. 

```


  return (
    <ScaleAppShell>
      <ScaleAppHeader slot="header" id="header">
        <div slot="menu-main">
          <ScaleNavMain name="Custom Item #1"></ScaleNavMain>
          <ScaleNavMain is-active name="Custom Item #3"></ScaleNavMain>
          <ScaleNavMain name="Custom Item #4"></ScaleNavMain>
        </div>
      </ScaleAppHeader>

      <p>some large content</p>
    </ScaleAppShell>
  );
};
```

**Comment by tommy-codez** [comment]
We want to change the <AppHeader> children depending on routing - this works fine except for the "stickiness" of the header.

I tested the code from my first exampe in different browsers but the stickiness doesnt work - unfortunately.

**Comment by nowseemee** [comment]
Hello everybody, thanks for all the input so far.

I made an attempt to reproduce the issue and this is what I came up with: https://codesandbox.io/s/create-react-app-forked-nos07

Looks like everything is in order. Please let me know how to break this. Thanks!

**Comment by tommy-codez** [comment]
Hi there,
it seems that the Framework we use (Tailwind CSS) break things with the CSS utilitiy classes...
I am not sure why the properties variant works. I will check further to find something workable..

Like to work with the APPHeader children component but it seems not to work for me :(.

**Comment by christopherGdynia** [comment]
Hi there,

the ```div.header``` gets a class sticky onScroll, tailwind gives it the sticky positioning, maybe renaming the class, or prefixing scale classes with ```scl-```

**Comment by tommy-codez** [comment]
Cool thanks @christopherGdynia - I will check it

**Comment by christopherGdynia** [comment]
Hello, 

is there any update?

Can you update the class div.header.sticky in Header component to div.header.header__sticky ? This should fix it.

**Comment by nowseemee** [comment]
Hello, yes, we are working on this. I post an update into this thread once the new version is published.

**Comment by nowseemee** [comment]
It's released: `3.0.0-beta.20` - I hope this works for Tailwind!
